.agents/modernize-explainer.md

```
@@ -0,0 +1,466 @@
+| `nuget.config` | NuGet sources | Standard config |
+| `.github/workflows/main.yml` | CI/CD pipeline | Add coverage, artifacts |
+
+---
+
+## Appendix C: Suppressed Analyzer Categories
+
+### Nullable (CS86xx) - 17 rules suppressed
+- CS8600, CS8601, CS8602, CS8603, CS8604, CS8605
+- CS8618, CS8619, CS8620, CS8625, CS8629
+- CS8764, CS8765, CS8766, CS8767, CS8769
+
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)